README.md

```diff
@@ -551,6 +551,14 @@ new_file_folder_path = ""
 # This is also imported from obsidian if not specified: specifically the option titled "New file location"
 daily_notes_folder = ""
 
+
+# Whether markdown links should include an extension or not
+# for example [File](file.md) or [File](file)
+include_md_extension_md_link = true
```

Comment:
Same here

Reply:
agreed, yeah. I'll do that now, one sec.

Reply:
I just changed that, I squashed the commit so it's the same message:

```
+ c2941e6...798653f include-extension -> include-extension (forced update)
```